SR2025/2025-04-23-competition-complete.md

```diff
@@ -0,0 +1,65 @@
+---
+to: Student Robotics 2025
+subject: Student Robotics 2025 comes to a close
+---
+
+![Competition Photo](https://studentrobotics.org/images/content/blog/sr2025/sr2025-photo.jpg TODO_IMAGE)
+
+Hi everyone,
+
+We'd like to thank you all for taking part in Student Robotics 2025 and making it such a successful event! An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/blog/2025-04-23-ths-win-sr2025/ TODO_BLOG_POST).
+
+Congratulations again to our winners:
+
+| Prize                           | Team
+|---------------------------------|--------------------------------
+| 1st Place                       | The Highfield School
+| 2nd Place                       | School of St Helen and St Katharine
+| 3rd Place                       | Ruthin School
+| Rookie Award                    | The Highfield School
+| Excellence in Engineering Award | Queen Mary’s College
+| Robot and Team Image            | School of St Helen and St Katharine
+| Online Presence                 | School of St Helen and St Katharine
+
+## Event Media
+
+If you'd like some mementos of the competition or would just like to see the photos which were taken, we've collected them all together into a photo album, which will be added to the blog post in the coming weeks.
+
+The livestreams of the matches are also available on YouTube, including footage from the final:
+
+* Day 1: https://www.youtube.com/watch?v=iJwuYluRUjE
+* Day 2: https://www.youtube.com/watch?v=m4DT0_MfAuk
+
+The in-venue soundtrack playlists can also be found on [the event page](https://studentrobotics.org/events/sr2025/competition/#soundtrack).
+
+## Feedback
```

Comment:
Add a deadline for submissions of the 18th May. That ensures that all submissions are in before we want to look at them on the w/c 19th.